ruff_lsp/server.py

```diff
@@ -955,39 +1032,25 @@ def _run_tool_on_document(
     else:
         argv += [document.path]
 
-    log_to_output(f"Running Ruff with: {argv}")
-    result: utils.RunResult = utils.run_path(
+    return run_path(
         argv=argv,
         use_stdin=use_stdin,
         cwd=settings["cwd"],
-        source=document.source.replace("\r\n", "\n"),
```

Comment:
Is it intentional that you removed the `\r\n` replacement? I don't think it is needed for ruff today and could, potentially, result in incorrect offsets.

Reply:
I asked charlie and he said it's likely historical. I removed it because i would like avoid doing any content transformations on the LSP side, those should all be done in ruff, and i'm a bit worried differences between the file in VS Code/on disk and what ruff thinks the file can get really confusing. For the offsets we communicate line and character in the LSP protocol, so i'd hope that they are unaffected